src/fhirtypes/ElementDefinition.ts

```diff
@@ -602,6 +615,192 @@ export class ElementDefinition {
     };
   }
 
+  fixValue(value: FixedValueType): void {
+    if (typeof value === 'boolean') {
+      this.fixBoolean(value);
+    } else if (typeof value === 'number') {
+      this.fixNumber(value);
+    } else if (typeof value === 'string') {
+      this.fixString(value);
+    } else if (value instanceof Code) {
+      this.fixFshCode(value);
+    } else if (value instanceof FshQuantity) {
+      this.fixFshQuantity(value);
+    } else if (value instanceof FshRatio) {
+      this.fixFshRatio(value);
+    }
+  }
+
+  private checkIfFixed(
+    value: boolean | string | number,
+    currentElementValue: boolean | string | number,
+    elementType: string
+  ): boolean {
+    if (currentElementValue != null && currentElementValue !== value) {
+      throw new PrimitiveValueAlreadyFixedError(value, elementType, currentElementValue);
+    }
+    return true;
+  }
+
+  private matchesRegex(value: string, regex: RegExp) {
```

Comment:
I don't know that this function should exist. It seems like it only handles the one operation within it that we could just use instead.

Reply:
Agreed.  I'm thinking maybe this function once did something else too?  But given its simple implementation, I'd just use the standard `/myregex/.test(value)` where it's needed.  If @ngfreiter did this based on conversation w/ me, then I apologize -- I probably did not express myself well.